### PR TITLE
radio double support

### PR DESCRIPTION
**Pull request by pelikhan**
Support for TNumber on the C++ layer. This protocol should be migrated in TS.

### CODE REVIEW

**Comment by abchatra** [comment]
This should go to next release, not today. Too late.

File: docs/reference/radio/packet.md [hunk 1]
```diff
@@ -10,6 +10,45 @@ A packet that was received by the radio.
...
 * `signal` - How strong the radio signal is. The exact range of values varies, but it goes approximately from `-128` dB (weak) to `-42` dB (strong).
 
+## Packet layout
+
+This section documents the data layout of the packet if you need to interpret the data outside of MakeCode.
+
+    Packet byte layout
+    | 0              | 1 ... 4       | 5 ... 8           | 9 ... 28
+    ----------------------------------------------------------------
+    | packet type    | system time   | serial number     | payload
```
Comment (mmoskal) [inline docs/reference/radio/packet.md]: Always sending time and serial seems somewhat wasteful for strings and buffers.

**Comment by mmoskal** [comment]
I'm also wondering, why don't we do all this stuff in TS? We could just have a function to send and receive a raw buffer and do the encoding in TS.

**Comment by pelikhan** [comment]
Tested on hardware. This code should definitely go to TS.

**Pull request merged**